regions/io/read_ds9.py

```diff
@@ -192,20 +194,33 @@ def ds9_region_list_to_objects(region_list):
             else:
                 raise ValueError("No central coordinate")
         else:
+            # Note: this should effectively never happen, because it would
```

Comment:
I think this error actually *can't* occur by user error.  If a user mistypes / enters an incorrect region type, the previous function, `ds9_string_to_region_list`, should fail.  Only regions that are parsed by `ds9_string_to_region_list` but do not have a corresponding entry within `ds9_region_list_to_objects` should reach this point, and that should only happen if someone updating this code forgets to add a new entry in this function.